src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform, color 0.3s;
```

Если в transition явно указать к каким параметрая ты устанавляваешь время, то время нужно прописывать после каждого параметра.
```suggestion
    transition: transform 0.3s, color 0.3s;
```